MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -21,12 +21,16 @@ namespace Microsoft.Xna.Framework.Content.Pipeline.Processors
     [ContentProcessor(DisplayName = "Sprite Font Description - MonoGame")]
     public class FontDescriptionProcessor : ContentProcessor<FontDescription, SpriteFontContent>
     {
+        [DefaultValue(true)]
+        public virtual bool PremultiplyAlpha { get; set; }

```

Comment:
The non-premultiplied case is why `CompressFontDXT3Block` reads and processes alpha and RGB separately.  Originally it used the alpha value for the RGB as well, but that is only valid if the texture is premultiplied.  For non-premultiplied, the RGB channels are all white and the glyph is defined by the alpha channel.